Header header
bool result
int8 shelfID
int8 potID
time pictureTime